modules/pipelines-web-console.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * layered-products-web-console.adc
+
+:_mod-docs-content-type: CONCEPT
+[id="pipelines-web-console_{context}"]
+= {pipelines-title} in the web console
+
+{pipelines-title} is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. You can use the OperatorHub in the {product-title} web console to install the {pipelines-title} Operator to create and modify pipeline objects.
```

Comment:
The double "to" does not work very well in my view. Yes, you can use OperatorHub in the web console to install Popelines. Then you can use the Pipelines view of the Developer web console to create pipelines and to interact with pipelines.